Scroller/EditableWrapper.js

```diff
@@ -311,10 +320,8 @@ const EditableWrapper = (props) => {
 			reset();
 
 			if (lastInputType === 'scroll') {
-				const offset = itemWidth * (lastMouseClientX > scrollContentCenter ? 1 : -1);
-
 				scrollContainerHandle.current.start({
-					targetX: scrollContentNode.scrollLeft + offset,
+					targetX: (scrollContentNode.scrollLeft + itemWidth * (lastMouseClientX > scrollContentCenter ? 1 : -1)) * (rtl ? -1 : 1),
```

Comment:
How about using a format similar to the below?

const offset = itemWidth * (!rtl ^ !(lastMouseClientX > scrollContentCenter) ? 1 : - 1);

Reply:
Yes it would be better to use format similar to the below. It's fixed.